pangeo-notebook/environment.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
  - esmpy
  - fastjmd95
  - fsspec
+ - gcm-filters
```

Comment:
looks like this should have an underscore: `gcm_filters`

Reply:
Thanks @scottyhq. I have been successfully installing using pip on Pangeo Cloud with a dash, so figured I could use that same syntax here.

Reply:
Just made the update to `gcm_filters`